src/app/shell/personal-cabinet/applications/applications.component.ts

```diff
@@ -35,6 +43,9 @@ export class ApplicationsComponent implements OnInit {
       this.applications = applications
     );
 
+    this.user$.subscribe(user => this.role = user.role);
```

Comment:
you can use selectSnapshot to avoid additional subscription

Reply:
@LenkVskv  see examples in these components 
![image](https://user-images.githubusercontent.com/27493959/120666188-123b9d00-c495-11eb-8663-fa061169b212.png)